README.md

```diff
@@ -42,8 +42,29 @@ Further, DoDiscover seeks to help the user feel confident with their assumptions
 
 ## Unite causal discovery and causal representation learning
 
-Causal representation learning is the task of learning high-level latent variables and the causal structure between them from low-level variables observed in data.
-DoDiscover seeks to support causal representation learning algorithms in the context of traditional causal discovery settings.
+DoDiscover is a Python library for causal discovery (causal structure learning).
+Our goal is to provide developers and researchers with guide rails for causal discovery that doesn't require deep knowledge of individual causal discovery algorithms.
+
+## What is the difference between dodiscover and causal-learn?
+
+The goal of dodiscover is to flatten the on-ramp to causal discovery algorithms.
+DoWhy provides a consistent API for various method for causal effect estimation.
```

Comment:
DoWhy goes beyond effect estimation with the structural causal models in the GCM module. What about "DoWhy provides a consistent API for various causal tasks, which typically require a graph structure."

Reply:
[570e5c0](https://github.com/py-why/dodiscover/pull/132/commits/570e5c0f8b663dd28404a7ea771cd16f5997d1ce)